### ISSUE TITLE
bug: error when using `xmlns:xlink`

### ISSUE THREAD

**Issue by gustavopch** [aidenybai/million#195]
**Describe the bug**
When trying to create an `<svg>` element with the `xmlns:xlink` attribute, the following error happens:
```
Uncaught DOMException: Failed to execute 'setAttributeNS' on 'Element': 'http://www.w3.org/1999/xlink' is an invalid namespace for attributes.
```

**To Reproduce**
1. Open https://stackblitz.com/edit/mega-millions-iqrsnb?file=index.html
2. See the error in the console

**Expected behavior**
It should be possible to create an `<svg>` element with the `xmlns:xlink` attribute just as it's possible to do this:
```js
document.body.innerHTML = '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"></svg>'
```

**Screenshots**
<img width="587" alt="image" src="https://user-images.githubusercontent.com/10941958/174188664-d69642e0-44c9-469f-a93f-044918b36f52.png">

**Device (please complete the following information):**
 - OS: macOS
- Browser: Chrome
- Version: 102

**Additional context**
https://github.com/aidenybai/million/blob/48353a144f65d21453cf430372fba9080f82dc01/src/million/createElement.ts#L32-L34

**Comment by cbbfcd** [comment]
[yep, more about this bug ](https://dom.spec.whatwg.org/#dom-element-setattributens).

**Comment by aidenybai** [comment]
The DOM-based repro is roughly this:

```js
const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
svg.setAttributeNS('http://www.w3.org/2000/svg', 'xmlns:xlink', 'http://www.w3.org/1999/xlink')
```

**Comment by cbbfcd** [comment]
@aidenybai if use `xlink:href` or others:

```js
// should like this???
const p = propName.split(':');
if (p[0] === 'xlink') {
  el.setAttributeNS(ns, p[1], value);
}
```

refer to [preact-svg](https://github.com/developit/preact-svg/blob/master/src/index.js#L104)

**Comment by aidenybai** [comment]
I see... I pushed a fix in d44a631. Thank you @cbbfcd @gustavopch!
